prime-router/metadata/hl7_mapping/common/datatype/xcn-contact.yml

```diff
@@ -18,14 +18,28 @@ elements:
     value: [ '%resource.name.given.tail()' ]
     hl7Spec: [ '%{hl7XCNFieldPath}-4' ]
 
-  - name: xcn-suffix
+  - name: xcn-suffix-from-value
+    condition: >
+      %resource.name.extension(%`rsext-name-suffix`).exists().not() and 
+      %resource.name.extension(%`rsext-name-suffix-degree`).exists().not() and 
+      %resource.name.extension(%`rsext-name-suffix-professional`).exists().not()
     value: [ '%resource.name.suffix.toString()' ]
     hl7Spec: [ '%{hl7XCNFieldPath}-5' ]
```

Comment:
I was trying to remove this element based on out conversation this morning, but it broke a handful of translation tests.. Not sure how to deal with it, but I'm leaning towards adding the extension to our sample fhir. Don't think there's an across-the-board sender transform we can use here since I'm sure the type of suffix depends on who the person is and what specific suffices we're dealing with.

Reply:
I'll maybe bring it up as another 17th tomorrow, and this'll just be slightly unresolved til then.

Reply:
Ended up handling this in [this commit](https://github.com/CDCgov/prime-reportstream/pull/11850/commits/daeb088120aae569baae65ea54229bbac0b05b4a). Instead of handling it with sender transforms, I just edited the FHIR samples (since they don't correspond to current actual senders).